balancer-js/src/modules/pools/pool-types/concerns/composableStable/exit.concern.ts

```diff
@@ -126,34 +125,31 @@ export class ComposableStablePoolExit implements ExitConcern {
           });
 
     const userData =
-      sortedValues.singleTokenOutIndex >= 0
+      sortedValues.singleTokenOutIndexWithoutBpt >= 0
         ? ComposableStablePoolEncoder.exitExactBPTInForOneTokenOut(
             bptIn,
-            sortedValues.singleTokenOutIndex as number
+            sortedValues.singleTokenOutIndexWithoutBpt
           )
         : ComposableStablePoolEncoder.exitExactBPTInForAllTokensOut(bptIn);
 
+    // MinAmounts needs a value for BPT for encoding
+    const minAmountsOutWithBpt = insert(
```

Comment:
Maybe we should keep the standard of having `minAmountsOut` as being "withBPT" and add "withoutBPT" to the one without it?
In other words, make `calcTokenOutGivenExactBptIn` return `minAmountsOutWithoutBpt` and `expectedAmountsOutWithoutBpt`.
This is not something critical, but it would be nice to keep a standard, so we don't have to always keep thinking: "does this have BPT or not"? 😅